### ISSUE TITLE
Malware: Pi Apps keeps trying to bundle new app installs with updates

### ISSUE THREAD

**Issue by jg00dman** [Botspot/pi-apps#2461]
### Cool ideas?

Pi Apps behaves like malware because it tries to install many apps the user has not requested along with updates.

**Comment by theofficialgman** [comment]
Please fill out under the proper issue template.

Pi-Apps does not install anything that the user did not request. During updates it refreshes any local install scripts with new versions as found on the GitHub if the user has selected that option from the update window (hint: read the tooltip for "Inactive updates". This is intentional. None of these inactive apps are installed against the users will.

**Comment by Botspot** [comment]
Pi-Apps stores all app install scripts locally. When we make improvements to one of those scripts, Pi-Apps will need to pull in those new changes. Of course, it could do this without asking permission as most app stores do, but we try to be extra transparent here and offer users the option to choose what to update.

TL;DR this is expected behavior that other app stores hide from you.
